src/plugins/paginate/_base.scss

```
@@ -24,6 +44,79 @@
 		border-bottom-right-radius: 4px;
 		border-top-right-radius: 4px;
 	}
+
+	li.active:nth-last-child(2) button {
+		border-bottom-right-radius: 4px;
+		border-top-right-radius: 4px;
+	}
+
+	.pagination {
+		& > li {
+			&.active {
+				& > button {
+					cursor: default;
+					outline-offset: -2px;
+				}
+			}
+			& > button {
+				background-color: #eaebed;
+				border: 1px solid #dcdee1;
+				color: #335075;
+				cursor: pointer;
+				display: inline-block;
+				float: left;
+				line-height: 1.4375;
+				margin-bottom: .5em;
+				margin-left: -1px;
+				padding: 10px 16px;
+				position: relative;
+				text-decoration: none;
+
+				&:hover {
+					@include hover-focus;
+				}
+				&:focus {
+					@include hover-focus;
+				}
+			}
+			&.disabled {
+				& + li {
+					& > button {
+						border-bottom-left-radius: $button-border-radius;
+						border-top-left-radius: $button-border-radius;
+					}
+				}
+			}
+
+			&:first-child {
+				& > button {
+					border-bottom-left-radius: $button-border-radius;
+					border-top-left-radius: $button-border-radius;
+					margin-left: 0;
+				}
+			}
+
+			&:last-child {
+				& > button {
+					border-bottom-right-radius: $button-border-radius;
+					border-top-right-radius: $button-border-radius;
+				}
+			}
```

The variable is already available in "`src/_variables.scss`".
```suggestion
			&.disabled {
				& + li {
					& > button {
						border-bottom-left-radius: $pager-border-radius;
						border-top-left-radius: $pager-border-radius;
					}
				}
			}

			&:first-child {
				& > button {
					border-bottom-left-radius: $pager-border-radius;
					border-top-left-radius: $pager-border-radius;
					margin-left: 0;
				}
			}

			&:last-child {
				& > button {
					border-bottom-right-radius: $pager-border-radius;
					border-top-right-radius: $pager-border-radius;
				}
			}
```